cmd/prober/prober.go

```diff
@@ -104,42 +108,46 @@ func encodeLevel() zapcore.LevelEncoder {
 }
 
 var (
-	logStyle       string
-	frequency      int
+	scPath      string
+	trPath      string
+	trustedRoot *root.TrustedRoot
+	staging     bool
+
+	frequency int
+	logStyle  string
+	addr      string
+	grpcPort  int
+
 	retries        uint
-	addr           string
-	rekorURL       string
-	fulcioURL      string
-	fulcioGrpcURL  string
-	tsaURL         string
 	oneTime        bool
 	runWriteProber bool
-	versionInfo    version.Info
+
+	rekorV1RequestsJSON string
+	fulcioRequestsJSON  string
```

Comment:
These also don't need to be moved into globals, they aren't used anywhere outside of `init()` so I would leave them where they were